### DISCUSSION TITLE
changes to the file custom-editor-style.scss are not compiled

### DISCUSSION THREAD

**Discussion by Matteo182** [understrap/understrap-child#336] [Q&A]
I am using corporate child theme, 
I started everything, npm intall and then npm run watch, everything seems to work, 
but when I edit the sass file, it starts, however my changes are not in the css file.

what's wrong? did i forget something?

**Comment by bacoords** [comment]
Can you send us the results of `node -v`?

**Reply by Matteo182** [reply]
v14.17.6

**Reply by bacoords** [reply]
We recommend 16. If you're using NVM, the switch is pretty easy, you just have to delete `node_modules` and run `npm install` again

**Reply by Matteo182** [reply]
i did, now i have version 16.0.0, i deleted node_module and i reinstalled everything with npm install, but the compilation of the sass still does not work

**Reply by Matteo182** [reply]
ok I solved it, I had to upgrade even if not, it signaled me a new version after upgrading node v7 to v8

thank you very much
